src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
-body {
+@import './variables';
+@import './product-card';
+@import './raiting-stars';
```

Comment:
There is a typo in the import statement for 'raiting-stars'. It should be 'rating-stars' to match the file name.